charts/streams-app/templates/deployment.yaml

```diff
@@ -58,7 +58,7 @@ spec:
       labels:
         app: {{ template "streams-app.name" . }}
         release: {{ .Release.Name }}
-        {{- range $key, $value := .Values.labels }}
+        {{- range $key, $value := .Values.podLabels }}
```

Comment:
Please add a default empty list to values.yaml

Reply:
There is also a condition above where you need to check podLabels instead

Reply:
> There is also a condition above where you need to check podLabels instead

I don't think there is a condition for Values.labels

Reply:
Sorry, you are right